docs/advanced/api/vitest.md

```diff
@@ -0,0 +1,521 @@
+
+::: warning
+As of Vitest 3, it's possible to have multiple test specifications with the same module ID (file path) if `poolMatchGlob` has several pools or if `typecheck` is enabled. This possibility will be removed in Vitest 4.
+:::
+
+```ts
+const specifications = await vitest.globTestSpecifications(['my-filter'])
+// [TestSpecification{ moduleId: '/tests/my-filter.test.ts' }]
+console.log(specifications)
+```
+
+## getRelevantTestSpecifications
+
+```ts
+function getRelevantTestSpecifications(
+  filters?: string[]
+): Promise<TestSpecification[]>
+```
+
+This method resolves every test specification by calling [`project.globTestFiles`](/advanced/api/test-project#globtestfiles). It accepts string filters to match the test files - these are the same filters that [CLI supports](/guide/filtering#cli). If `--changed` flag was specified, the list will be filtered to include only files that changed. `getRelevantTestSpecifications` doesn't run any test files.
+
+::: warning
+This method can be slow because it needs to filter `--changed` flags. Do not use it if you just need a list of test files.
+
+- If you need to get the list of specifications for known test files, use [`getModuleSpecifications`](#getmodulespecifications) instead.
+- If you need to get the list of all possible test files, use [`globTestSpecifications`](#globtestspecifications).
+:::
+
+## mergeReports
+
+```ts
+function mergeReports(directory?: string): Promise<TestRunResult>
+```
+
+Merge reports from multiple runs located in the specified directory (`--merge-reports` if not specified). This value can also be set on `config.mergeReports` (by default, it will read `.vitest-reports` folder).
+
+Note that the `directory` will always be resolved relative to the working directory.
+
+This method is called automatically by [`startVitest`](/advanced/guide/tests) if `config.mergeReports` is set.
+
+## collect
+
+```ts
+function collect(filters?: string[]): Promise<TestRunResult>
+```
+
+Execute test files without running test callbacks. `collect` returns unhandled errors and an array of [test modules](/advanced/api/test-module). It accepts string filters to match the test files - these are the same filters that [CLI supports](/guide/filtering#cli).
+
+This method resolves tests specifications based on the config `include`, `exclude`, and `includeSource` values. Read more at [`project.globTestFiles`](/advanced/api/test-project#globtestfiles). If `--changed` flag was specified, the list will be filtered to include only files that changed.
+
+::: warning
+Note that Vitest doesn't use static analysis to collect tests. Vitest will run every test file in isolation, just like it runs regular tests.
+
+This makes this method very slow, unless you disable isolation before collecting tests.
+:::
+
+## start
```

Comment:
`start` is twice on this page.

Reply:
Ah, it's the old one - thanks, I removed it